Wright_Will/Assignments/OOP/Human/Ninja.rb

```diff
@@ -0,0 +1,17 @@
+require_relative "Human"
+
+class Ninja < Human
+    def initialize
+        @stealth = 175
+        self
+    end
+    def steal(attackee)
+        self.atack(attackee)
```

Comment:
Does this work? I spy a typo. Also see what happens when you just call attack(attackee)